index.js

```diff
@@ -3,7 +3,7 @@
  */
 
 import {AppRegistry} from 'react-native';
-import App from './App';
+import {App} from './App';
```

Comment:
Somewhere you write `import {SomeComponent} from '...'` and somewhere `import { SomeComponent } from '...'`, the style of imports should be consistent